test/envs/volsync.yaml

```diff
@@ -10,7 +10,6 @@ templates:
     driver: $vm
     container_runtime: containerd
     network: $network
-    cni: kindnet
```

Comment:
This change should really be in another commit, I included it here since it is not very important.